questions/explain_scrum_to.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "Explain Scrum to ..."
+parent: Katas
+---
+Explain Scrum to:
+- a 5 year old;
+- a 90 year old;
+- someone outside of software development.
```

Comment:
Though it doesn't fit PSM III questions, I thought it might be a nice exercise in a community of practice :).